### ISSUE TITLE
Add support for elasticsearch

### ISSUE THREAD

**Issue by droid-mohit** [DrDroidLab/PlayBooks#81]
Allow someone to run lucene queries within a playbook.

**Comment by droid-mohit** [comment]
Development of this feature can be tracked here: https://github.com/DrDroidLab/PlayBooks/pull/283

**Comment by droid-mohit** [comment]
Related PRs:
1. New connector PR: https://github.com/DrDroidLab/PlayBooks/pull/280
2. New Task Source PR: https://github.com/DrDroidLab/PlayBooks/pull/282/files
3. Asset manager PR: https://github.com/DrDroidLab/PlayBooks/pull/284

**Comment by sidPhoenix17** [comment]
Was completed after #291
